### PR TITLE
Fix failing tests

### LINKED ISSUE #5115: Randomly failing test: `tests/plugins/image2/link`
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Open `tests/plugins/image2/link#tests%2Fplugins%2Fimage2%2Flink%20test%20link%20bare%20widget%20(link%20dialog)%3A%20inline%2C%20none`

### Expected result

Test passed

### Actual result

Test Failed
FT

## Other details

* Browser: IE9
* OS: Windows
* CKEditor version: 4.18.0(pre-relased; builded)
* Installed CKEditor plugins: …

### LINKED ISSUE #5116: Failing tests in `/tests/plugins/templates/templates` (Chrome Mobile)
### Reproduction steps

1. Run tests on presets
2. Open: http://0.0.0.0:1030/tests/plugins/templates/templates

### Expected result

All tests passed

### Actual result

Tests are failing:
- http://0.0.0.0:1030/tests/plugins/templates/templates#tests%2Fplugins%2Ftemplates%2Ftemplates%20test%20insert%20template%20file%20and%20replace%20content
- http://0.0.0.0:1030/tests/plugins/templates/templates#tests%2Fplugins%2Ftemplates%2Ftemplates%20test%20insert%20template%20file%20at%20cursor%20position
- http://0.0.0.0:1030/tests/plugins/templates/templates#tests%2Fplugins%2Ftemplates%2Ftemplates%20test%20inserting%20template%20file%20makes%20dialog%20busy

image

## Other details

* Browser: Chrome Mobile (99.0.4844.58)
* OS: Android
* CKEditor version: 4.18.0 (pre-released, presets, full all)
* Installed CKEditor plugins: full all

### LINKED ISSUE #5119: tests/core/skin/manual/moonomultiselect should be skipped on mobile
## Type of report

Task

## Provide description of the task

There are three new manual tests that should be skipped on mobiles, as it is not possible to achieve the expected result, as dialog hides after blurring it.

Tests:
```
http://localhost:1030/tests/core/skin/manual/kamamultiselect
http://localhost:1030/tests/core/skin/manual/moonolisamultiselect
http://localhost:1030/tests/core/skin/manual/moonomultiselect
```

### PR DESCRIPTION
**Pull request by sculpt0r**
## What is the purpose of this pull request?

Bug fix / Failinig tests fix
## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [X] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

```
none
```

## What changes did you make?

*Give an overview…*
For #5119 - added test ignore
For #5116 - enlarged test awaiting time. On slower networks (mobile) the template was still loading when the test executed the assertion. 
For #5115 - I added those tests to ignored on IE9. On group and on the individual runs they failed 1/10...

## Which issues does your PR resolve?

Closes #5119
Closes #5116 
Closes #5115

### CODE REVIEW

File: tests/plugins/image2/link.js [hunk 1]
```diff
@@ -296,6 +296,11 @@
 		// -- Link bare with data ---------------------------
 
 		'test link bare widget (widget.setData): inline, none': function() {
+			// (#3761, #5115)
```
[the remark this answers is not in the document]
Comment (Comandeer) [reply tests/plugins/image2/link.js:299]: Additionally I'm not able to make this test fail for me in IE9 on `master` :thinking:

**Review by Comandeer — CHANGES_REQUESTED** [review]
LGTM, I just have some doubts around the `image2/link` test (see my inline comments).

File: tests/plugins/image2/link.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sculpt0r) [reply tests/plugins/image2/link.js:299]: Where did you test it? BS or VM?

**Comment by sculpt0r** [comment]
@Comandeer - the reference issue number is corrected. Good catch 👍  Could you take a look at this PR again? Also: https://github.com/ckeditor/ckeditor4/pull/5143#discussion_r846025400 - I was testing it at VM - what environment did you use?

**Comment by Comandeer** [comment]
> I was testing it at VM - what environment did you use?

I've also tested int on VM.

**Comment by sculpt0r** [comment]
As for failing tests on IE9: This time the test fails at 8-th refresh:
image2_link

**Review by Comandeer — APPROVED** [review]
Ok, if this test really fails in IE9 (still no luck on my VM), then I'm ok with these changes.

**Pull request merged**